Focus on Scientific Python (#152)
scientific-python/scientific-python.org: content/en/community/ways-to-contribute.md

-->

## 5 ways to contribute to Scientific Python without coding

<!--
Hello everyone, I’m Juanita.
Welcome to the Scientific Python videos! Today I will be sharing with you some ways in which you can contribute to open source software projects without actually having to code.
-->

> Scientific Python software is code designed in a way that it is accessible to the public, everyone can view, modify, and distribute it easily, according to the terms of permissive software licenses.

Reviewer: Maybe also something along the lines of:

"All projects have a license that determine what you can and cannot do; for Scientific Python these licenses are typically very straightforward: you may use and modify the software, as long as you give credit to the original authors."

Author: See what you think about what I did.

It relies on peer review and community production.
This is why you can engage in the Scientific Python community easily and contribute in any way that you feel comfortable.

### Issue testing and triaging

> Every Scientific Python project has its own issue tracker where users post bugs, UX improvements, or any other issues they are having.

Reviewer: ```suggestion
Every Scientific Python project has its own issue tracker where users post bugs, suggest UX improvements, and report other issues they are having.
```
